styleguide-app/Examples/AssignmentIcon.elm

```diff
@@ -53,5 +53,20 @@ example =
                 [ ( "startPrimary", AssignmentIcon.startPrimary )
                 , ( "startSecondary", AssignmentIcon.startSecondary )
                 ]
+            , IconExamples.view "Activities"
+                [ ( "assessment", AssignmentIcon.assessment )
+                , ( "practice2", AssignmentIcon.practice2 )
+                , ( "standards", AssignmentIcon.standards )
+                , ( "writing", AssignmentIcon.writing )
+                ]
+            , IconExamples.view "Practice & Assessment"
+                [ ( "assignArrow", AssignmentIcon.assignArrow )
+                , ( "home", AssignmentIcon.home )
```

Comment:
"Home" seems more like a UIIcon rather than an AssignmentIcon, to me, even if it's only used in the lizard so far